config/brokers/mt-channel-broker/deployments/controller.yaml

```diff
@@ -54,13 +54,13 @@ spec:
           - name: METRICS_DOMAIN
             value: knative.dev/eventing
 
-          # Due to the trivial per-Broker cost, we inject Brokers into every
+          # Due to the trivial per-Broker cost, we _can_ inject Brokers into every
           # namespace by default. To change this default simply change this
-          # to "false".  To opt namespaces out of Broker injection, label
+          # to "true".  To opt namespaces out of Broker injection, label
           # them with:
           #    knative-eventing-injection: disabled
```

Comment:
Why not always creating a broker (channel or mt, doesn't matter) when and only when that label is present?

Reply:
no change on the label - that just generally prevents _that_ namespace to have an injected broker